scripts/update-values-schema-json.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Generate json schema for chart values.
+#
+# https://github.com/karuppiah7890/helm-schema-gen
+
+set -e -o pipefail [[ -n "$DEBUG" ]] && set -x
+
+chart_dirs=$(ls charts)
+for chart_dir in ${chart_dirs}; do
+  echo "schema-gen charts/${chart_dir} chart..."
+  helm schema-gen "charts/${chart_dir}/values.yaml" >| "charts/${chart_dir}/values.schema.json"
+done
```

Comment:
Knowing how to install `helm-schema-gen` can be tedious, so it would be nice to just run `update-values-schema-json.sh`. Therefore, I suggest how to execute it in a container and get generated `values.schema.json` files.

- https://github.com/superbrothers/helm-charts/commit/49ffab15b0abe525fab30e3ceb5a6e249c6c9074

Reply:
@superbrothers 
I have corrected the pointing out issues :bow: 
Please check it out.
https://github.com/scalar-labs/helm-charts/pull/62/commits/9bb1402f7572c652bc4b41be28c9b9808c3fe0fc